azavea/osm-state-of-the-data
# Move react to subproject and bump dependencies

## Overview

Dependencies were pretty up to date, only a few minor bugfix bumps. 

Looks like we missed a few dotfiles when we migrated this repo, this PR also adds them back. 

Adding STRTA, tests and other things are for separate issues. This PR just gets the react app out of the way.

## Testing Instructions

```
yarn install --pure-lockfile
yarn start
```
React app should compile and serve at localhost:3000. The default configuration doesn't show tiles.

Closes #3
